modules/ROOT/pages/upgrade-path.adoc

```diff
@@ -0,0 +1,35 @@
+= Upgrade Runtime Fabric Components
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Runtime Fabric contains a collection of services which run within the cluster, including the agent used to connect to Anypoint Platform. Perform the following steps to upgrade these services.
```

Comment:
```suggestion
Anypoint Runtime Fabric 1.3 and 1.4 contain a collection of services that run within the cluster, including the agent used to connect to Anypoint Platform. To upgrade these services, you must meet some prerequisites, understand deployed application availability, apply the upgrade, and perform troubleshooting if necessary.
```
Suggested rewording is an example of making sure that the container topic text introduces contained subtopics.